CMakeLists.txt

```
@@ -9,27 +9,41 @@ endif()
 
 find_package(Kodi REQUIRED)
 find_package(p8-platform REQUIRED)
-find_package(OpenGL REQUIRED)
 
 find_package(ProjectM REQUIRED)
 
 add_definitions(-DHAS_SDL_OPENGL)
 
-set(INCLUDES ${OpenGL_INCLUDE_DIR}
-             ${KODI_INCLUDE_DIR}
+set(INCLUDES ${KODI_INCLUDE_DIR}
              ${p8-platform_INCLUDE_DIRS}
              ${PROJECTM_INCLUDE_DIRS})
 
 if(CORE_SYSTEM_NAME STREQUAL "linux")
-  set(DEPLIBS ${OPENGL_LIBRARIES}
-              -Wl,--whole-archive ${PROJECTM_LIBS} -Wl,--no-whole-archive
+  set(DEPLIBS -Wl,--whole-archive ${PROJECTM_LIBS} -Wl,--no-whole-archive
               ${extra_libs})
 else()
-  set(DEPLIBS ${OPENGL_LIBRARIES}
-              ${PROJECTM_LIBS}
+  set(DEPLIBS ${PROJECTM_LIBS}
               ${extra_libs})
 endif()
 
+if(APP_RENDER_SYSTEM STREQUAL "gl" OR NOT APP_RENDER_SYSTEM)
+  find_package(OpenGL)
```

Make this package required and remove the found check below.
```suggestion
  find_package(OpenGL REQUIRED)
```